core/src/main/java/org/jdbi/v3/core/statement/StatementContext.java

```diff
@@ -546,6 +571,38 @@ public void setExceptionMoment(Instant exceptionMoment) {
         this.exceptionMoment = exceptionMoment;
     }
 
+    /**
+     * Retrieve the number of mapped rows. Only intended for internal instrumentation to call.
+     */
+    @Alpha
```

Comment:
The setters are `@Beta` and the getters are `@Alpha`?

Reply:
Yeah, the Alpha ones are really 'never call' internal things, and the Beta ones are actually intended to be API. I wish I could structure this differently, but the Java package visibility model doesn't really allow friend classes. Maybe we should introduce an `@Internal` annotation.